pkg/piperutils/FileUtils.go

```diff
@@ -47,3 +49,30 @@ func Copy(src, dst string) (int64, error) {
 	nBytes, err := io.Copy(destination, source)
 	return nBytes, err
 }
+
+// FindFiles ...
+func FindFiles(root string, pattern string) ([]string, error) {
+
+	var files []string
+
+	r, err := regexp.Compile(pattern)
+	if err != nil {
+		return files, err
+	}
+
+	err = filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+
+		if err != nil {
+			return err
+		}
+
+		if !info.IsDir() && r.MatchString(info.Name()) {
```

Comment:
This matches only the base-name of the file (see https://golang.org/pkg/os/#FileInfo). What I expected when seeing this PR was that it matches glob-patterns. Since this is also a frequent need in the pipeline, what do you think about implementing that "instead"?

Reply:
there is use of glob patterns already here: https://github.com/SAP/jenkins-library/blob/master/cmd/checkmarxExecuteScan.go#L23